src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -1352,7 +1363,7 @@ public void shouldIncrementCommonMetrics() {
     @Test
     public void shouldCallUpdateCookieMetricsWithExpectedValue() {
         // given
-        given(bidderCatalog.isActive(any())).willReturn(false);
+        given(bidderCatalog.isActive(any())).willReturn(true, false);
```

Comment:
I've changed it because now we call `isActive` two times in our `holdAuction` flow - first (the one I've added) in `extractBidderRequests`, second, the one that was before, - in `updateRequestMetric` so I thought to leave the order as it was. But removing this stub and using the default stub from set up `true` looks to pass as well (is it good though?), so I guess this stub is redundant after all.